Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -55,7 +55,10 @@ - (void)updateMask
         return;
     }
     maskedSubview.frame = self.bounds;
-    [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:maskedSubview isOutgoing:self.isOutgoing];
+    // The JSQ masks are RTL-safe, so we need to invert the
```

Comment:
typo? `s/are RTL-safe/are not RTL-safe/`

It is a good day when you get to xor.

Reply:
Done.